### PR TITLE
PR for release 1.3.0

### PR DESCRIPTION
**Pull request by apeltzer**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/nanostring/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/nanostring _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,24 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.2.2 - Nanometre patch
```
Comment (christopher-mohr) [inline CHANGELOG.md:6]: Don't forget to add the date before release

**Review by christopher-mohr — CHANGES_REQUESTED** [review]
I will continue later with the rest.

File: nextflow.config [hunk 2]
```diff
@@ -39,23 +39,32 @@ params {
...
+    config_profile_url         = null
 
     // Max resource options
     // Defaults only, expecting to be overwritten
     max_memory                 = '128.GB'
     max_cpus                   = 16
     max_time                   = '240.h'
 
-     // Config options
+    // Config options
```
Comment (christopher-mohr) [inline nextflow.config:67]: duplicated

**Review by christopher-mohr — APPROVED** [review]
LGTM

**Review by charles-plessy — CHANGES_REQUESTED** [review]
Overall it looks good to me but I note that you have not changed the version number in |<https://github.com/nf-core/nanostring/blob/dev/assets/nf-core_nanostring_tubemap.png>… maybe it is better to do that before merging the PR? (I also note a typo after "Complex" that you can fix at the same time.)
image

**Comment by christopher-mohr** [comment]
> Overall it looks good to me but I note that you have not changed the version number in |https://github.com/nf-core/nanostring/blob/dev/assets/nf-core_nanostring_tubemap.png… maybe it is better to do that before merging the PR? (I also note a typo after "Complex" that you can fix at the same time.) image

well spotted :) thank you!

**Review by charles-plessy — APPROVED** [review]
Looks good !

**Pull request merged**